src/config/passport.conf.js

```diff
@@ -8,6 +8,8 @@ import User from '../app/models/user.model.js';
 import md5 from 'md5';
 import randomstring from 'randomstring';
 
+const reservedUsernames = ['yuttadhammo', 'doug', 'martin', 'robin', 'sebastian', 'it'];
+
 export default (passport) => {
   // Define length boundaries for expected parameters
   let bounds = {
```

Comment:
The minimum email length of 5 characters prevents someone with a shorter username to login.